Description: Since Epiphany is not our default browser,
 allow users to uninstall it using the GNOME Software app
Author: Jeremy Bicha <jbicha@debian.org>
---
 data/org.gnome.Epiphany.appdata.xml.in |    1 -
 1 file changed, 1 deletion(-)

--- a/data/org.gnome.Epiphany.appdata.xml.in
+++ b/data/org.gnome.Epiphany.appdata.xml.in
@@ -32,7 +32,6 @@
     <kudo>UserDocs</kudo>
   </kudos>
   <project_group>GNOME</project_group>
-  <compulsory_for_desktop>GNOME</compulsory_for_desktop>
   <project_license>GPL-2.0+</project_license>
   <developer_name>The GNOME Project</developer_name>
   <url type="bugtracker">https://bugzilla.gnome.org/enter_bug.cgi?product=epiphany</url>
